pkg/ipam/multipool.go

```diff
@@ -251,14 +247,39 @@ func newMultiPoolManager(conf Configuration, nodeWatcher nodeWatcher, owner Owne
 	}
 
 	// Subscribe to CiliumNode updates
-	nodeWatcher.RegisterCiliumNodeSubscriber(c)
+	c.eventloop(node)
```

Comment:
Perhaps call this `startCiliumNodeEventLoop` so it's clear it's not blocking and what it's handling.